business_requirement_deliverable/models/business.py

```diff
@@ -236,7 +236,7 @@ class BusinessRequirement(models.Model):
     total_revenue = fields.Float(
         compute='_compute_deliverable_total',
         string='Total Revenue',
-        store=False
+        store=True
```

Comment:
@sudhir-serpentcs This change is duplicated with https://github.com/OCA/business-requirement/pull/105/files#diff-005420dbf67a67f7fbc633822129feceL280

Please remove from this PR. The rest LGTM

cc @elicoidal